### ISSUE TITLE
Baby Chest cannot be opened

### ISSUE THREAD

**Issue by richardhendricks** [GTNewHorizons/NewHorizons#2160]
#### Which modpack version are you using?
2.0.1.3
#
#### What did you try to do, and what did you expect to happen?
I wanted to store a stack of cells in 1 slot, so I tried crafting  a baby chest.  It doesn't open.  A one slot one stack container can be useful for stuff.  I was going to use it to pull off and store 1 stack of biodiesel from my biodiesel cell pipe, so I always have some for my LV diesel gens.  I currently use a tank for this, but that means the cells don't back up until the tank is full. Overflow cells are carried to the cetane misxer.
#
#### What happend instead? (Attach screenshots if needed)
Chest cannot open
#
#### What do you suggest instead/what changes do you propose?
Fix chest?

**Comment by Methes** [comment]
You can still pump items in and out of the chest, it just doesn't have any GUI.

**Comment by Ugachaga** [comment]
IIRC, Dirt Chest has 1 slot and working GUI.

**Comment by richardhendricks** [comment]
Thanks for the tip, I'll look into the dirt chest.

**Comment by Dream-Master** [comment]
I guess Babychest only for decoration @Namikon ?

**Comment by Namikon** [comment]
No, it is (was) a fully functional chest. I haven't touched the code for uhm.. 2 years? But it **does** have a GUI, and even a specialized opening sound. The Hitbox is a lot smaller; So make sure you're in range

**Comment by Dream-Master** [comment]
No it not opened

![image](https://user-images.githubusercontent.com/2970688/33481391-8aeb9cfc-d694-11e7-9d70-0da94f51c9c8.png)
I can see the hitbox @Namikon

**Comment by Namikon** [comment]
Well... then someone must've broken it since then. I have no time to look up the cause for this, so the chest will remain "decorative" for the time being.

**Comment by Dream-Master** [comment]
NP not a big deal.

**Comment by richardhendricks** [comment]
@Ugachaga Dirt chest works, but only takes 1 stack of dirt.

@Dream-Master Not sure why this was closed?

**Comment by Dream-Master** [comment]
because Namikon dont have the time to look into it or fix it.

**Comment by richardhendricks** [comment]
Where is the source repo for it?

**Comment by Dream-Master** [comment]
in the core mod
package com.dreammaster.modbabychest;

**Comment by richardhendricks** [comment]
Mini-chest is an alternative for now.
